arangodb-net-standard/GraphApi/GraphApiClient.cs

```diff
@@ -87,5 +87,24 @@ public async Task<GetGraphResponse> GetGraphAsync(string graphName)
                 throw await GetApiErrorException(response);
             }
         }
+
+        /// <summary>
+        /// Lists all edge collections within this graph.
+        /// GET /_api/gharial/{graph}/edge
+        /// </summary>
+        /// <param name="graphName"></param>
+        /// <returns></returns>
+        public async Task<GetGraphEdgesResponse> GetGraphEdgesAsync(string graphName)
```

Comment:
I propose we name this `GetGraphEdgeCollectionsAsync` and the corresponding response model `GetGraphEdgeCollectionsResponse`.  Otherwise it makes it seem like the method would return the edge documents themselves.

Reply:
Similarly we'd need to update the test method names